README.md

```diff
@@ -6,7 +6,7 @@ Ghost is a free, open, simple blogging platform. Visit the project's website at
 
 To get your own Ghost blog running on Heroku, click the button below:
 
-[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/cobyism/ghost-on-heroku)
+[![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/ns11/ghost-on-heroku)
```

Comment:
Since this change is being sent from your fork, I don’t have access to make changes (since you’re the owner of that repo). Do you mind changing it back?